com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -21,7 +20,6 @@
 using MLAPI.Transports.Tasks;
 using MLAPI.Messaging.Buffering;
 using Unity.Profiling;
-using UnityEditor.VersionControl;
```

Comment:
this was causing a compile error and the using directive is not even needed at all :)

```
.../NetworkManager.cs(24,19): error CS0234: The type or namespace name 'VersionControl' does not exist in the namespace 'UnityEditor' (are you missing an assembly reference?)
```